.github/workflows/Crane.yaml

```diff
@@ -76,7 +76,7 @@ jobs:
           target: google_apis
           arch: x86
           disable-animations: true
-          script: ./gradlew connectedCheck --stacktrace
+          script: ./gradlew connectedAndroidDebugTest --stacktrace
```

Comment:
Looks like this change causes Crane builds to fail https://github.com/android/compose-samples/actions/runs/8898242654/job/24435886140?pr=1380